docs/UserGuide.adoc

```diff
@@ -10,7 +10,7 @@ ifdef::env-github[]
 :tip-caption: :bulb:
 :note-caption: :information_source:
 endif::[]
-:repoURL: https://github.com/se-edu/addressbook-level4
+:repoURL: https://github.com/CS2103T-T09-B1/main
```

Comment:
You should update the features in the User guide, few important points:
- Should include v2.0 features, not just current features
- Un-implemented features should be marked ‘coming in v2.0’ or something like that.